dhctl/pkg/kubernetes/actions/deckhouse/install.go

```diff
@@ -156,11 +156,42 @@ func UnlockDeckhouseQueueAfterCreatingModuleConfigs(kubeCl *client.KubernetesCli
 	})
 }
 
-func AddReleaseChannelToDeckhouseModuleConfig(kubeCl *client.KubernetesClient, cfg *config.DeckhouseInstaller) error {
+func ConfigureReleaseChannel(kubeCl *client.KubernetesClient, cfg *config.DeckhouseInstaller) error {
+	// if we have correct semver version we should create Deckhouse Release for prevent rollback on previous version
+	// if installer version > version in release channel
+	if tag, found := config.ReadVersionTagFromInstallerContainer(); found {
+		deckhouseRelease := unstructured.Unstructured{}
+		deckhouseRelease.SetUnstructuredContent(map[string]interface{}{
+			"apiVersion": "deckhouse.io/v1alpha1",
+			"kind":       "DeckhouseRelease",
+			"metadata": map[string]interface{}{
+				"name": tag,
+			},
+			"spec": map[string]interface{}{
+				"version": tag,
```

Comment:
Shouldn't we check, that tag is semver compatible. Otherwise Deckhouse will fail

Reply:
checked here
https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/config/deckhouse_config.go#L138

Reply:
we use this function here. Inside function we check tag on semver compatibility if not then we return `false` from function